roles/openshift_node/defaults/main.yml

```diff
@@ -1,6 +1,6 @@
 ---
 os_firewall_allow:
-- service: OpenShift kubelet
+- service: Kubernetes kubelet
```

Comment:
:+1: much better naming for this.

As an aside, the way that firewall allows is happening now is horrible, I'm ashamed of the way I did it, since it is really inflexible...  I need to check on firewalld support from upstream (docker, kubernetes, and openshift itself), then we can do away with this os_firewall monstrosity and "require" firewalld and use the firewalld ansible module.